build.gradle

```diff
@@ -2,7 +2,7 @@ import edu.wpi.first.gradlerio.deploy.roborio.RoboRIO
 
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2022.1.1"
+    id "edu.wpi.first.GradleRIO" version "2022.2.1"
```

Comment:
For now please revert back to the older version, 2022.1.1

Reply:
Please don't make irrelevant changes in this branch.